ios/MullvadVPN.xcodeproj/xcshareddata/xcschemes/MullvadVPN.xcscheme

```diff
@@ -88,7 +88,7 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
-      launchStyle = "1"
+      launchStyle = "0"
```

Comment:
FYI: Should we keep this launch style?

Reply:
I have been meaning to keep it at `0` (start the app automatically & attach the debugger), but somehow I managed to commit `1` (wait for the app to launch before attaching debugger). This basically reverts the change to what I think is a good default.